src/main/resources/assets/js/app/wizard/ContentWizardPanel.ts

```diff
@@ -1246,17 +1246,20 @@ export class ContentWizardPanel
         return new ContentWizardDataLoader().loadDefaultModels(site, this.contentType.getContentTypeName()).then(
             defaultModels => {
                 this.defaultModels = defaultModels;
-                return this.initPageModel(this.liveEditModel, defaultModels).then(pageModel => {
-                    const livePanel = this.getLivePanel();
-                    const needsReload = !this.isSaving(); // pageModel is updated so we need reload unless we're saving already
-                    if (livePanel) {
-                        livePanel.setModel(this.liveEditModel);
-                        if (needsReload && reloadPage) {
-                            this.debouncedEditorRefresh(true);
-                        }
-                    }
-                    return needsReload;
-                });
+                const livePanel = this.getLivePanel();
```

Comment:
Why did you move this line out of the `then` handler? It won't be used if `this.liveEditModel` is undefined, so I don't see why we need it here.

Reply:
You are right. Moved it inside the `then` scope.